### ISSUE TITLE
Entity Framework 6 Support

### ISSUE THREAD

**Issue by LaikaGlove** [KorzhCom/EasyData#184]
This looks like a wonderful alternative to the old Dynamic Data from .NET Framework.  Any plans to add support for Entity Framework 6?

**Comment by antifree** [comment]
Hello. 

Thank you for the idea!

But I think EF 6 is quite outdated technology and is not commonly used in current projects. I suppose that with EF 6 support we will need to add old ASP.NET support as well, so it will require implementation for two new packages EasyData.AspNet and EasyData.EF6 which will almost duplicate existing functionally and will force us to spend more effort on supporting both versions. 

If we will see interest from many developers in it, I will find some time to implement it. Until that I would keep this feature request on hold. What do you think about it, @korzh?

**Comment by korzh** [comment]
Agree. EF6 is like 10 years old now. We would add support for it only if there is a much higher demand.
